backend/services/cloudkit_web_client.py

```
@@ -0,0 +1,326 @@
+    def query_user_records(self, record_type: str, user_id: str, limit: int = 100) -> List[Dict]:
+        """Query records for a specific user"""
+        try:
+            # For UserProfile, query by record name (which is the user ID)
+            if record_type == 'UserProfile':
+                try:
+                    record = self.fetch_record(user_id)
+                    if record:
+                        return [self._parse_cloudkit_fields(record)]
+                    else:
+                        return []
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)